packages/react/src/components/internal/TextFieldWrapper/TextFieldWrapper.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render as testingRender, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { spy } from 'sinon';
+import { axe } from 'jest-axe';
+
+import TextFieldWrapper, { TextFieldWrapperProps } from './';
+
+type RenderProps = Partial<TextFieldWrapperProps> & {
+  [key: string]: any;
+};
+
+const render = ({ className, children, ...otherProps }: RenderProps = {}) =>
+  testingRender(
+    <TextFieldWrapper className={className} {...otherProps}>
+      {children || <p>Children</p>}
+    </TextFieldWrapper>
+  );
+
+test('should render children', () => {
+  render();
+  expect(screen.getByText('Children')).toBeInTheDocument();
+});
+
+test('should render TextFieldWrapper with default className', () => {
+  render();
+  expect(screen.getByText('Children').parentElement).toHaveClass(
+    'TextFieldWrapper'
+  );
+});
+
+test('should render TextFieldWrapper with custom className', () => {
+  render({ className: 'banana' });
+  expect(screen.getByText('Children').parentElement).toHaveClass(
+    'TextFieldWrapper',
+    'banana'
+  );
+});
+
+test('should render TextFieldWrapper with onClick', async () => {
+  const onClick = spy();
+  render({ onClick });
+  await userEvent.click(screen.getByText('Children'));
+  expect(onClick.calledOnce).toBe(true);
+});
```

Comment:
This seems like an odd test to add for this component.

Reply:
That's fair. I wanted to include it because the usage of `ComboBox` which this replaced passed an `onClick`, so wanted to make sure there's a test confirming that that would still work. I can remove it if you feel like it's unneccessary.

Reply:
As long as it's covered by the Combobox tests, we can omit it here.